docs/search/criteria_reference/createdat_criterion.md

```diff
@@ -4,11 +4,13 @@ The `CreatedAt` Search Criterion searches for products based on the date when th
 
 ## Arguments
 
-- `createdAt` - indicating the date that should be matched, provided as a `DateTimeInterface` object
+- `createdAt` (PHP), `created_at` (REST) - indicating the date that should be matched, provided as a `DateTimeInterface` object
 - `operator` - Operator constant (EQ, GT, GTE, LT, LTE)
```

Comment:
Same problem of PHP vs REST here. While PHP can use constants, REST must use [the values of those constants](https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Query/Criterion/Operator.php#L18-L22).

Maybe something like:

```suggestion
- `operator` - Operator constant (EQ, GT, GTE, LT, LTE) in PHP or its value in REST
```

Reply:
The examples are so naturally talkative in comparison.